### DISCUSSION TITLE
Automate Browsing without Plugin?

### DISCUSSION THREAD

**Discussion by bweinlich-wps** [danny-avila/LibreChat#2071] [Feature Requests & Suggestions]
With a ChatGPT Plus account, the web is automatically searched for certain questions without the need to activate a specific plugin or assistant. This way, questions about the current weather or knowledge questions about movies or music albums are answered much more accurately and with virtually no hallucinations.

The web browsing plugins integrated in LibreChat unfortunately do not yet work as reliably.

Is there a plan to optimize the web browsing capability of LibreChat to reduce hallucinations or to integrate it to automatically use browsing when the built in knowledge of the LLM is nit sufficient?

**Comment by danny-avila** [comment]
Using Assistants and/or plugins and the upcoming change to system-wide model settings, you could basically have whatever preset/tools/assistant preloaded for this functionality.
